pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -203,6 +234,40 @@ func (p *staticPolicy) RemoveContainer(s state.State, containerID string) error
 	return nil
 }
 
+func (p *staticPolicy) inheritCPUs(s state.State, pod *v1.Pod, numCPUs int) (cpuset.CPUSet, error) {
+	// Get a sorted slice of all init Container cpuSets.
+	var cpuSets []*cpuset.CPUSet
+	for _, container := range pod.Spec.InitContainers {
+		containerID, err := p.containerMap.Get(pod, &container)
+		if err != nil {
+			continue
+		}
+		if cset, ok := s.GetCPUSet(containerID); ok {
+			cpuSets = append(cpuSets, &cset)
+		}
+	}
+	sort.Slice(cpuSets, func(i, j int) bool {
```

Comment:
Nit... if we only want to find the largest cpuSet, I think we can do that in `O(n)` by not sorting and just finding the max element.

Reply:
Fixed